doc/radosgw/multisite.rst

```diff
@@ -43,44 +43,43 @@ Production <https://access.redhat.com/documentation/en-us/red_hat_ceph_storage/3
 Functional Changes from Infernalis
 ==================================
 
-In Kraken, you can configure each Ceph Object Gateway to
-work in an active-active zone configuration, allowing for writes to
-non-master zones.
-
-The multi-site configuration is stored within a container called a
-"realm." The realm stores zone groups, zones, and a time "period" with
-multiple epochs for tracking changes to the configuration. In Kraken,
-the ``ceph-radosgw`` daemons handle the synchronization,
-eliminating the need for a separate synchronization agent. Additionally,
-the new approach to synchronization allows the Ceph Object Gateway to
-operate with an "active-active" configuration instead of
-"active-passive".
+In Kraken, each Ceph Object Gateway can be configured to work in an
+active-active zone configuration, allowing for writes to non-master zones.
+
+The multi-site configuration is stored within a container called a "realm". The
+realm stores zone groups, zones, and a time "period" with multiple epochs for
+tracking changes to the configuration. In Kraken, the ``ceph-radosgw`` daemons
+handle the synchronization, which eliminates the need for a separate
+synchronization agent. Additionally, the new approach to synchronization allows
+the Ceph Object Gateway to operate with an "active-active" configuration
+instead of "active-passive".
 
 Requirements and Assumptions
 ============================
 
-A multi-site configuration requires at least two Ceph storage clusters,
-preferably given a distinct cluster name. At least two Ceph object
-gateway instances, one for each Ceph storage cluster.
+A multi-site configuration requires at least two Ceph storage clusters. Each of
+the Ceph storage clusters should have a distinct cluster name. The multi-site
```

Comment:
We no longer encourage non-default cluster names and I strongly suspect one can no longer create one

Reply:
I've entirely removed the sentence beginning "Each of..".